dnSpy/dnSpy/Themes/wpf.styles.templates.xaml

```diff
@@ -2085,6 +2085,56 @@
             </Trigger>
         </Style.Triggers>
     </Style>
+
+    <Style x:Key="SortableColumnHeaderStyle"
+           BasedOn="{StaticResource {x:Type GridViewColumnHeader}}"
+           TargetType="{x:Type GridViewColumnHeader}">
+        <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+    </Style>
+
+    <DataTemplate x:Key="SortableColumnHeaderTemplate">
+        <DataTemplate.Resources>
+            <Style TargetType="Path">
+                <Setter Property="Margin" Value="10,0,0,0" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+                <Setter Property="StrokeThickness" Value="1" />
+                <Setter Property="Fill" Value="{DynamicResource HeaderDefaultText}" />
+            </Style>
+        </DataTemplate.Resources>
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
```

Comment:
Would it be possible to move the arrow closer to the name so it's easier to see the arrow when reading the name (eg. using Width=Auto here and Width=* below? Or will the arrow disappear if there's not enough room for both the text and arrow?